build/karma.conf.babel.js

```diff
@@ -42,23 +42,19 @@ module.exports = (karmaConfig) => {
     },
     webpack: {
       devtool: 'cheap-module-source-map',
-      module: {
-        ...webpackConfig.module,
+      module: Object.assign({}, webpackConfig.module, {
```

Comment:
Man, I could have sworn it was though I'm on `6.2.1` and Node swears otherwise.  Could be I need a flag or `use strict` or some other something enabled.